### ISSUE TITLE
vmalert crashes afret upgrade from v1.35.3-cluster to v1.35.4-cluster

### ISSUE THREAD

**Issue by stress-t** [VictoriaMetrics/VictoriaMetrics#473]
After upgrading from vmalert-v1.35.3-cluster to vmalert-v1.35.4-cluster, vmalert crashes when starting and analyzing rules

```
vmalert      | 2020-05-13T09:57:29.492Z info    VictoriaMetrics/app/vmalert/manager.go:77       reading alert rules configuration file from "/etc/vmalert/alerts/*.yaml"
vmalert      | panic: runtime error: invalid memory address or nil pointer dereference
vmalert      | [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x7c1621]
vmalert      | 
vmalert      | goroutine 1 [running]:
vmalert      | main.(*Rule).Restore(0xc0001c2270, 0xa6cbe0, 0xc0000b6d40, 0x0, 0x0, 0x34630b8a000, 0x7fab21d0b108, 0x0)
vmalert      |  github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/rule.go:303 +0x171
vmalert      | main.(*Group).Restore(0xc000082eb0, 0xa6cbe0, 0xc0000b6d40, 0x0, 0x0, 0x34630b8a000, 0x10, 0xc0000abcf0)
vmalert      |  github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/group.go:42 +0x107
vmalert      | main.(*manager).startGroup(0xc0000b5880, 0xa6cbe0, 0xc0000b6d40, 0xc000018760, 0x12, 0xc000102400, 0x3f, 0xc00000f820, 0x3, 0x3, ...)
vmalert      |  github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/manager.go:61 +0x244
vmalert      | main.(*manager).update(0xc0000b5880, 0xa6cbe0, 0xc0000b6d40, 0xc000095020, 0x1, 0x1, 0x101, 0x0, 0x3e8)
vmalert      |  github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/manager.go:105 +0x88c
vmalert      | main.(*manager).start(...)
vmalert      |  github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/manager.go:46
vmalert      | main.main()
vmalert      |  github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/main.go:93 +0x560
```

**Version**
```
$ ./vmalert-prod --version
vmalert-20200512-203449-tags-v1.35.4-cluster-0-g3b0f66a2
```

**Used command-line flags**
      - '--rule=/etc/vmalert/alerts/*.yaml'
      - '--remotewrite.url=http://vminsert:8480/insert/0/prometheus'
      - '--datasource.url=http://vminsert:8480/select/0/prometheus'
      - '--notifier.url=http://10.10.10.7:9093'

**Comment by hagen1778** [comment]
Thanks for report!

**Comment by hagen1778** [comment]
Hi @stress-t! Could you pls check the updated version in https://github.com/VictoriaMetrics/VictoriaMetrics/commit/415b1ddfb57a030205e4616dd4b42779ecbc8f3e

Could you build vmalert from this commit and verify whether it works as expected? 
To build dev version:
1. Install Go. The minimum supported version is Go 1.13. 
2. Run `make vmalert` from the root folder of the repository. It builds vmalert binary and puts it into the bin folder.

To build prod version:
1. Install docker.
2. Run `make vmalert-prod` from the root folder of the repository. It builds `vmalert-prod` binary and puts it into the bin folder.

**Comment by stress-t** [comment]
Hi @ hagen1778
I rebuilt vmalert-prod from 415b1dd commit
vmalert-prod starts normally

**Comment by hagen1778** [comment]
Thanks for confirmation!

**Comment by valyala** [comment]
FYI, the bugfix has been included in [v1.35.5](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
